go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/spf13/pflag v1.0.7
 	github.com/spf13/viper v1.20.1
 	github.com/stretchr/testify v1.10.0
-	github.com/zeta-chain/protocol-contracts v1.0.2-athens3.0.20250728071033-90592fb858dd
+	github.com/zeta-chain/protocol-contracts v0.0.0-20250909184950-6034c08e5870
```

Comment:
this is pointing to protocol-contracts main branch commit with this change, repo was migrated, and there is no tag like 1.0.2-athens or similar, should we update something here?